charter.html

```diff
@@ -270,20 +270,18 @@ <h2>Success Criteria</h2>
       Recommendation Snapshot</a>, and <a href="https://www.w3.org/2021/Process-20211102/#candidate-recommendation-draft">Candidate
       Recommendation Draft</a>. The WG does not intend to publish specifications as <a href="https://www.w3.org/2021/Process-20211102/#RecsPR">Proposed Recommendations</a>.
 
-          <p>To reach the Candidate Recommendation Snapshot stage, each normative
-            specification is expected to have <a href="https://www.w3.org/Consortium/Process/#implementation-experience">at
-              least two independent implementations</a> of every feature defined in the specification.
-            Interoperability of implementations will be verified by passing open test suites.
+    <p>There should be testing plans for each specification, starting from the earliest drafts.</p>
 
-          <p>Each normative specification should contain separate sections detailing security and privacy implications for implementers, Web authors, and end users.</p>
-
-          <p>There shall be a testing plan and open test suite developed that covers all functional
-          aspects of normative specifications developed by the WG.
-
-    <p>Normative specifications which have user-facing features should contain a section on accessibility that describes the benefits and impacts, including ways specification features can be used to address them, and recommendations for maximizing accessibility in implementations.</p>
-          
-    <p>In considering features, the group will state, during the development of those features or in the text of the document, privacy implications. Proposals should clearly state privacy issues, how they intend to address those issues, and the advertising use cases addressed in each deliverable.</p>          
-        </section>
+    <p>To promote interoperability, all changes made to specifications in Candidate Recommendation or to features that have deployed implementations should have tests. Testing efforts should be conducted via the Web Platform Tests project.</p>
+    
+    <p>Each specification should contain sections detailing security and privacy implications for implementers, Web authors, and end users, as well as recommendations for mitigation. There should be a clear description of the residual risk to the user or operator of that protocol after threat mitigation has been deployed.</p>
+    
+    <p>Each specification should contain a section on accessibility that describes the benefits and impacts, including ways specification features can be used to address them, and recommendations for maximizing accessibility in implementations.</p>
+    
+    <p>All new features should be supported by at least two intents to implement before being incorporated in the specification.</p>
```

Comment:
"at least two intents to implement" is a lower bar than "two independent implementations", and I think we want to keep insisting that things in living standards without two actual implementations are marked "at risk" or called out in some other way. @cwilso / @tantek, any thoughts? We should probably make the same comment on https://w3c.github.io/charter-drafts/charter-template.html.

("two intents to implement" is also awkward phrasing that echoes an old Chromium practice that's not in use anymore ... but it's clear enough what it means.)

Reply:
@jyasskin The conversations we have had made it clear that we don't want to exceed the standard requirements of the charter. @cwilso, @tantek, @jyasskin if you want this changed from the default language of the template, My understanding is the changes you need have to start at the template before they come to this charter.

Reply:
I'm sure we'll be doing that at TPAC (https://github.com/w3c/tpac2023-breakouts/issues/43), but since the charter template is Team-defined and never gets an AC vote, the charters themselves are when AC members tend to give feedback. +1 that we ought to find a way to do it earlier.

Can you share who's pushing to avoid extra requirements on what goes into the CRs? We shouldn't make you carry messages between us.

Reply:
@jyasskin this change comes out of conversations with @dontcallmedom but also from the thread reviewing the charter which notes a need to sync with the template - https://github.com/w3c/strategy/issues/222

Reply:
Thanks. I've also filed an issue with the template at https://github.com/w3c/charter-drafts/issues/436. Don't let my concern here slow you down on merging this PR. Hopefully we can find some better wording for the template and then merge it here in a separate PR.

Reply:
```suggestion
    <p>All new features should have expressions of interest from at least two potential implementors before being incorporated in the specification.</p>
```